MovingBlocks/TerasologyLauncher
# Extract About tab controller

### Contains

- extract AboutView controller and FMXL
- refactor AboutViewController to make better use of Optional and Stream
- add Google Guava library to dependencies

Closes #465.
Part of #462.

### How to test

Just check out this branch and start the launcher from source. You should be able to see content in the _About_ tab (again).

This is is due to the fact that JavaFX will call `initialize()` on the controller, which will set up the static content. 

### Outstanding before merging

- [x] add JavaDoc for `AboutViewController`
